Source/Utils/WarpXUtil.cpp

```diff
@@ -398,6 +399,28 @@ void ReadBCParams ()
                 }
             }
         }
+#ifdef WARPX_USE_PSATD
+        int maxwell_solver_id = GetAlgorithmInteger(pp_algo, "maxwell_solver");
+        if (maxwell_solver_id == MaxwellSolverAlgo::PSATD) {
+            ParmParse pp_psatd("psatd");
+            int do_moving_window_input = 0;
+            pp_warpx.query("do_moving_window", do_moving_window_input);
+            if (do_moving_window_input == 1) {
```

Comment:
This is just a very small detail, but I wonder if it would be more straightforward to call the `int` variable simply `do_moving_window` (wihtout the extra `_input`), as the input parameter itself.

The reason why being that I think it could be a bit less confusing for someone who reads the code to read this
```cpp
pp_warpx.query("do_moving_window", do_moving_window);
```
rather than this
```cpp
pp_warpx.query("do_moving_window", do_moving_window_input);
```
I have seen in the past people being possibly confused by the latter query and maybe wrongly setting `do_moving_window_input` in their input file, thinking that it is the correct input name, for instance after quickly grepping through the code.

But anyways it is just a small suggestion.

```suggestion
            int do_moving_window = 0;
            pp_warpx.query("do_moving_window", do_moving_window);
            if (do_moving_window == 1) {
```